core/src/subgraph/instance.rs

```diff
@@ -26,10 +27,11 @@ where
         // we use the same order here as in the subgraph manifest to make the
         // event processing behavior predictable
         let manifest_id = manifest.id;
+        let spec_version = Version::parse(&manifest.spec_version)?;
```

Comment:
Could we have a constant with the maximum supported version for this node, and check it here? So that in the future we don't have old nodes trying to run newer spec versions.

Reply:
I've included a version requirement that ensures the subgraph being deployed is using a spec version equal to or below the current Graph-Node spec version. 

However, I don't love having it buried in `core/src/subgraph/instance`.  It could be added as part of the runtime host builder to be passed into `from_manifest()` and eventually be used to version other aspects of the subgraph instance.